### PR TITLE
fix pgsql connection issues from missing quotes in selfoss config file

### PR DESCRIPTION
**Pull request by srccon**
I found that some of the generated password characters can cause issues with selfoss connecting to the pgsql database, adding quotes in the config fixes this issue.

### CODE REVIEW

File: roles/news/templates/var_www_selfoss_config.ini.j2 [hunk 1]
```diff
@@ -1,9 +1,10 @@
 ; see https://github.com/SSilence/selfoss/wiki/Configuration for more information about the configuration parameters
 [globals]
 db_type=pgsql
-db_database={{selfoss_db_database}}
-db_username={{selfoss_db_username}}
-db_password={{selfoss_db_password}}
+db_host=127.0.0.1
```
Comment (mikeashley) [inline roles/news/templates/var_www_selfoss_config.ini.j2]: Why this change?

File: roles/news/templates/var_www_selfoss_config.ini.j2 [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
+db_database="{{selfoss_db_database}}"
```
Comment (mikeashley) [inline roles/news/templates/var_www_selfoss_config.ini.j2]: Do you think quoting the database and username are necessary?  Just trying to focus the patch on the specific problem and not make speculative changes.

File: roles/news/templates/var_www_selfoss_config.ini.j2 (hunk 1 above)
Comment (srccon) [inline roles/news/templates/var_www_selfoss_config.ini.j2]: Technically it's not necessary, I posted up what I had changed to get it to work. I like to over specify.

File: roles/news/templates/var_www_selfoss_config.ini.j2 (hunk 2 above)
Comment (srccon) [inline roles/news/templates/var_www_selfoss_config.ini.j2]: I'm not sure what the escape character are, commas certainly. Prevents against escape characters causing issues in the username as well, in case a user randomly generates their username.

Comment (srccon) [inline roles/news/templates/var_www_selfoss_config.ini.j2]: Database, probably not.

File: roles/news/templates/var_www_selfoss_config.ini.j2 (hunk 1 above)
Comment (srccon) [inline roles/news/templates/var_www_selfoss_config.ini.j2]: It defaults to localhost anyway, this would make it easier for someone to specify a separate db server.

File: roles/news/templates/var_www_selfoss_config.ini.j2 (hunk 2 above)
Comment (fengor) [inline roles/news/templates/var_www_selfoss_config.ini.j2]: quoting those is considered good form. I'd +1 those changes

File: roles/news/templates/var_www_selfoss_config.ini.j2 (hunk 1 above)
Comment (neuhaus) [inline roles/news/templates/var_www_selfoss_config.ini.j2]: The default is "localhost", not 127.0.0.1. This change could mean that the database connection uses a TCP socket instead of a UNIX domain socket which is twice as fast. Please either change it to localhost or remove it.

**Comment by mikeashley** [comment]
I removed the `db_host` setting and merged the (squash) commit.  Thanks for the contribution!
